alvr/client_core/build.rs

```diff
@@ -53,5 +53,6 @@ fn main() {
         for path in cpp_paths {
             println!("cargo:rerun-if-changed={}", path.to_string_lossy());
         }
+        println!("cargo:rerun-if-changed={}", "cpp/bindings.h");
```

Comment:
Isn't this already included in the instruction above?

Reply:
Nah it seems like we are only watching the .CPP files

Reply:
Not quite, we are watching everything in the cpp folder, so .cpp and .h files.

Reply:
Oh ! I dunno why but unless i add this line the changes in bindings.h were not reflected in bindings.rs, maybe a bug for another day ig XD